example/index.html

```diff
@@ -18,6 +18,9 @@
     <script>
         var options = {
             html5: {
+                // hlsjsPluginConfig: {
+                //     enable: false, // default true 
+                // },
```

Comment:
I'd say move this into `hlsjsConfig` to make it similar to `contrib-hls` config. I guess `hls.js` can handle some rogue config fine and if it is not fine you can always "block" this out when the `hlsjsConfig` clone is handled.